packages/interface/src/capability.ts

```diff
@@ -405,6 +427,13 @@ export interface UnavailableProof extends Failure {
   readonly link: UCANLink
 }
 
+export interface DIDResolutionError extends Failure {
+  readonly name: 'DIDResolutionError'
+  readonly did: UCAN.DID
+
+  readonly cause?: Unauthorized
```

Comment:
Is it really important that the `cause` always be `Unauthorized` and not another Error?